### PR TITLE
Update summit NFS server names

### PR DESCRIPTION
**Pull request by athornton**

### CODE REVIEW

File: services/nublado2/values-summit.yaml [hunk 1]
```diff
@@ -55,20 +55,20 @@ nublado2:
       - name: latiss
         nfs:
           path: /repo/LATISS
-          server: atarchiver.cp.lsst.org
+          server: auxtel-atarchiver.cp.lsst.org
```
Comment (ktlim) [inline services/nublado2/values-summit.yaml:58]:
```suggestion
          server: auxtel-archiver.cp.lsst.org
```

File: services/nublado2/values-summit.yaml [hunk 2]
```diff
@@ -55,20 +55,20 @@ nublado2:
...
       - name: base-auxtel
         nfs:
           path: /lsstdata/base/auxtel
-          server: atarchiver.cp.lsst.org
+          server: auxtel-atarchiver.cp.lsst.org
```
Comment (ktlim) [inline services/nublado2/values-summit.yaml:62]:
```suggestion
          server: auxtel-archiver.cp.lsst.org
```

**Review by ktlim — APPROVED** [review]
Couple of typos.

File: services/nublado2/values-summit.yaml (hunk 2 above)
Comment (athornton) [reply services/nublado2/values-summit.yaml]: Got em.  Thanks.

**Pull request merged**